### PR TITLE
update limit-related rules to allow limits.d

### PR DESCRIPTION
**Pull request by shawndwells**
Update XCCDF to accept limits.d

### CODE REVIEW

**Comment by shawndwells** [comment]
Hmm, the OpenShift tests are failing:

`````
error: could not run steps: step src failed: could not wait for build: the build src failed after 33s with reason DockerBuildFailed: Docker build strategy has failed.

{"command":"git checkout master","component":"clonerefs","...itched to branch 'master'\n","time":"2020-04-15T18:41:36Z"}
{"command":"git fetch https://github.com/ComplianceAsCode/...0/head -\u003e FETCH_HEAD\n","time":"2020-04-15T18:41:36Z"}
{"command":"git merge --no-ff a70d129ea9032d229d6bee1876d2...ot something we can merge\n","time":"2020-04-15T18:41:36Z"}
{"component":"clonerefs","error":"one or more of the recor..."msg":"Failed to clone refs","time":"2020-04-15T18:41:36Z"}
error: build error: running 'umask 0002 && /clonerefs && f... -perm -0775 | xargs -r chmod g+xw' failed with exit code 1
`````
https://storage.googleapis.com/origin-ci-test/pr-logs/pull/ComplianceAsCode_content/5600/pull-ci-ComplianceAsCode-content-master-e2e-aws-moderate/148/build-log.txt

Will retest after the CaC Jenkins finishes.

File: linux_os/guide/system/permissions/restrictions/coredumps/group.yml [hunk 1]
```diff
@@ -10,7 +10,8 @@ description: |-
     also contain sensitive information, or unnecessarily occupy large
     amounts of disk space.
     <br /><br />
-    Once a hard limit is set in <tt>/etc/security/limits.conf</tt>, a
+    Once a hard limit is set in <tt>/etc/security/limits.conf</tt>, or
+    to a file within the <tt>/etc/security/limits.d</tt> directory, a
```
Comment (shawndwells) [inline linux_os/guide/system/permissions/restrictions/coredumps/group.yml:14]:
```suggestion
    to a file within the <tt>/etc/security/limits.d/</tt> directory, a
```

**Comment by matejak** [comment]
LGTM, even the OVAL takes that into the account, so the XCCDF really needed this update.

**Pull request merged**